swap_meet/vendor.py

```diff
@@ -1,2 +1,70 @@
+from swap_meet.item import Item
 class Vendor:
-    pass
+    def __init__ (self,inventory= None ):
```

Comment:
careful with syntax! Even tho it didn't break, it is not standard practice to have a space between the function name and the parentheses
```suggestion
    def __init__(self,inventory = None):
```